api/search.py

```diff
@@ -157,6 +159,8 @@ def as_keywords(t):
                 lneg = term.startswith("not:")
             if cond == "complete":
                 q = Q(is_complete=True)
+            elif cond == "pull":
+                q = Q(subject__contains='[PULL') | Q(subject__contains='[GIT PULL')
```

Comment:
OK. We can live with subject maching for now, until an "is_pull" is introduced somewhere: there will be more places where we should special handling them (like using "git fetch" instead of "git am").